fiddler/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# CHANGELOG - fiddler
+
+## 1.0.0 / 2022-05-07
```

Comment:
For consistency with other listings, please use `yyyy-mm-dd` format. I believe this should be `2022-07-05`